packages/algob/test/internal/deployer.ts

```diff
@@ -677,3 +678,92 @@ describe("DeployerDeployMode", () => {
 		assert.deepEqual(res.assetDef, asaDef);
 	});
 });
+describe("Helper functions", () => {
+	const env = mkEnv("network1");
+	const deployerCfg = new DeployerConfig(env, new AlgoOperatorDryRunImpl());
+	let deployer: DeployerDeployMode;
+	let alice: algosdk.Account;
+	let bob: algosdk.Account;
+
+	beforeEach(() => {
+		deployer = new DeployerDeployMode(deployerCfg);
+		alice = generateAccount();
```

Comment:
```suggestion
		[alice,bob] = generateAccount();
```
Maybe you can do something like this.

Reply:
this just returns one account. I need to invoke it twice